pkg/okteto/namespace.go

```diff
@@ -110,3 +121,20 @@ func DeleteNamespace(ctx context.Context, namespace string) error {
 	var body DeleteBody
 	return query(ctx, q, &body)
 }
+
+func validateNamespace(namespace string) error {
+	if len(namespace) > MAX_ALLOWED_CHARS {
```

Comment:
could we use a function from client-go instead of doing the validation ourselves?

Reply:
If we do that we can not get which of the validation failed. The method is validateDNS1123 which checks the length and the regex, but if we want to return which is the one failing we will have to check each one